core/src/test/java/com/yahoo/ycsb/measurements/exporter/TestMeasurementsExporter.java

```diff
@@ -52,7 +52,7 @@ public void testJSONArrayMeasurementsExporter() throws IOException {
         JsonNode  json = mapper.readTree(out.toString("UTF-8"));
         assertTrue(json.isArray());
         assertEquals(json.get(0).get("measurement").asText(), "Operations");
-        assertEquals(json.get(3).get("measurement").asText(), "MaxLatency(us)");
-        assertEquals(json.get(11).get("measurement").asText(), "5");
+        assertEquals(json.get(4).get("measurement").asText(), "MaxLatency(us)");
+        assertEquals(json.get(11).get("measurement").asText(), "4");
```

Comment:
My additional measure is moving the indices of the JSONArrayMeasurementsExporter. The build also works with the following line.
```
assertEquals(json.get(12).get("measurement").asText(), "5");
```